src/routes/images.rs

```diff
@@ -78,10 +78,11 @@ async fn thumbnail(
   let (size, file) = parts.into_inner();
 
   let url = format!(
-    "{}/image/thumbnail{}/{}",
+    "{}/image/process.{}?src={}&thumbnail={}",
```

Comment:
Maybe we should also change the Lemmy API so that it stays identical to the Pict-rs API. Or maybe not, I dont know. This is certainly the easiest solution.

Reply:
Yeah I'd definitely prefer it to match the new pict-rs API exactly.

Reply:
it would be easier to expose webp vs jpg in the client if the lemmy api matched pict-rs, but you should also consider how you could limit the lemmy api to do exactly what you need and nothing more

Reply:
@Nutomic we currently have:

```
    .service(web::resource("/pictrs/image/{filename}").route(web::get().to(full_res)))
    .service(
      web::resource("/pictrs/image/thumbnail{size}/{filename}").route(web::get().to(thumbnail)),
    )
```

How about we change the thumbnail to take in a format, and add another route that does full resolution, but with a format:

```
    .service(web::resource("/pictrs/image/{filename}").route(web::get().to(full_res)))
    .service(
      web::resource("/pictrs/image/{filename}/thumbnail/{size}/format/{format}).route(web::get().to(thumbnail)),
    .service(
      web::resource("/pictrs/image/{filename}/format/{format}).route(web::get().to(full_res_with_format)),
    )
```

Reply:
Sounds fine to me, though I feel like using URL parameters like `?format=jpg&thumbnail=128` would be cleaner.